### PR TITLE
[Jobs API] Describe Triggered Job Handling Assumptions

### PR DESCRIPTION
**Pull request by cicoyle**
add specific logic for what assumptions are made for triggered jobs for http, grpc, sdks

### CODE REVIEW

**Review by JoshVanL — APPROVED** [review]

File: daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md [hunk 1]
```diff
@@ -94,6 +94,73 @@ In this example, at trigger time, which is `@every 1s` according to the `Schedul
...
+### Triggered Job Handling Assumptions
+
+#### HTTP
+
+When you create a job using Dapr's Jobs API, Dapr will automatically assume there is an endpoint available at 
+`/job/<job-name>`. For instance, if you schedule a job named `test`, Dapr expects your application to listen for job 
+events at `/job/test`. Ensure your application has a handler set up for this endpoint to process the job when it is 
+triggered. For example:
+
+```go
```
Comment (yaron2) [inline daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md:106]: Can you add a note that his example uses Go but is applicable to any language?

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md (hunk 1 above)
Comment (cicoyle) [reply daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md:106]: sure thing

Comment (cicoyle) [reply daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md:106]: done. lmk if you want me to tweak it

Comment (yaron2) [reply daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md:106]: Thanks, I ended up changing it directly. We'll soon need to edit this again anyway as Java/Dotnet support is coming :)

**Review by yaron2 — APPROVED** [review]

File: daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md (hunk 1 above)
Comment (WhitWaldo) [reply daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md:106]: Added .NET bits with #4385 

I don't know if the very bottom section is needed calling out the various SDK approaches given that we'll have just listed them in the above codetabs.

**Pull request merged**